charts/prometheus-elasticsearch-exporter/templates/podmonitoring.yaml

```
@@ -0,0 +1,50 @@
+{{- if .Values.podMonitoring.enabled }}
```

might be useful to also check for the capability?

```suggestion
{{- if and .Values.podMonitoring.enabled .Capabilities.APIVersions.Has "monitoring.googleapis.com/v1" }}
```

---

I was able to test this with `helm template . -f values.yaml -f ci/pod-monitoring-values.yaml -a "monitoring.googleapis.com/v1"`. However, I don't see a way to set this in a values.yaml for CI validation.